telegram/ext/_conversationhandler.py

```diff
@@ -676,11 +676,14 @@ def _schedule_job(
         try:
             # both job_queue & conversation_timeout are checked before calling _schedule_job
             j_queue = application.job_queue
-            self.timeout_jobs[conversation_key] = j_queue.run_once(
-                self._trigger_timeout,
-                self.conversation_timeout,  # type: ignore[arg-type]
-                data=_ConversationTimeoutContext(conversation_key, update, application, context),
-            )
+            if j_queue:
+                self.timeout_jobs[conversation_key] = j_queue.run_once(
+                    self._trigger_timeout,
+                    self.conversation_timeout,  # type: ignore[arg-type]
+                    data=_ConversationTimeoutContext(
+                        conversation_key, update, application, context
+                    ),
+                )
```

Comment:
interesting that mypy fails for you locally, since it's not failing in the CI. if you add `--no-verify` to `git commit …` you can skip the pre-commit hooks. Let's see what the CI thinks